### PR TITLE
Support Sig v3

### PR DESCRIPTION
**Pull request by jmatsu**
- The latest buildtools is required to support min sdk 33 or more
- Key rotated signing prints multiple Signer but they do not have number! And also, the first Signer is what we need.
- Addressed rubocop issues

### CODE REVIEW

**Comment by jmatsu** [comment]
ref: https://engineering.linecorp.com/ja/blog/air-go-apk-signing

File: Gemfile [hunk 1]
```diff
@@ -4,3 +4,23 @@ source "https://rubygems.org"
 
 # Specify your gem's dependencies in android_apk.gemspec
 gemspec
+
+# General ruby development
+gem "rake", "~> 10.0"
```
Comment (jmatsu) [inline Gemfile:9]: rubocop recommends all development gems are defined in Gemfile itself.

File: Gemfile.lock [hunk 2]
```diff
@@ -8,94 +8,109 @@ GEM
   remote: https://rubygems.org/
```
Comment (jmatsu) [inline Gemfile.lock:8]: These gem updates don't affect to the runtime app.

File: lib/android_apk.rb [hunk 3]
```diff
@@ -6,10 +6,10 @@
 require "tmpdir"
 require "zip"
 
-require_relative "./android_apk/app_icon"
-require_relative "./android_apk/error"
-require_relative "./android_apk/resource_finder"
-require_relative "./android_apk/xmltree"
+require_relative "android_apk/app_icon"
```
Comment (jmatsu) [inline lib/android_apk.rb:9]: `rubocop issue`

File: lib/android_apk.rb [hunk 4]
```diff
@@ -348,7 +348,7 @@ def backward_compatible_adaptive_icon?
   def self._parse_values_workaround(str)
     return nil if str.nil?
 
-    str.scan(/^'(.+)'$/).map { |v| v[0].gsub(/\\'/, "'") }
+    str.scan(/^'(.+)'$/).map { |v| v[0].gsub("\\'", "'") }
```
Comment (jmatsu) [inline lib/android_apk.rb:351]: ditto

File: spec/fixture/signatures/signature-v3/README.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+```
```
Comment (jmatsu) [inline spec/fixture/signatures/signature-v3/README.md:1]: These keystores can be public because they are only for development.

**Review by satsukies — APPROVED** [review]
LGTM ✅

**Pull request merged**